book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
### Your Attr
```
  attr_accessor :title, :author
  attr_reader :rentals
```
### Expected Attr from Previous Reviewer

```
  attr_accessor :title, :author, :rentals
```

- [ ] Kindly make your attr as suggested by the previous reviewer :blush: